.github/workflows/validate.yml

```diff
@@ -187,7 +187,7 @@ jobs:
         run: go install golang.org/x/tools/cmd/goimports@latest
 
       - name: beautysh
-        run: pip install beautysh
+        run: pipx install beautysh && beautysh --version
 
       - name: cljstyle
         run: curl -sLO https://raw.githubusercontent.com/greglook/cljstyle/main/util/install-cljstyle && chmod +x install-cljstyle && sudo ./install-cljstyle
```

Comment:
I'm surprised to see a sudo in GitHub workflow

locally ok, why not, but on workflow, it's strange

Yes, I know it's not a line you are changing :P

Reply:
the remark would be the same for the 3 next items

Reply:
Just used the same as for my local install 😮

Reply:
Just removed the sudos, hopefully it doesn't break anything

Reply:
It was 😮 

https://github.com/hougesen/mdsf/actions/runs/8555780971/job/23444049400?pr=170

Reply:
It would have sounded strange to me otherwise